int8 num_tiles_x
int8 num_tiles_y
float64 map_length_x
float64 map_length_y
float64[] heightmap